NFT-RFC-002-use-cases.md

```diff
@@ -411,12 +411,52 @@ Uncertainty: “How sure are we about the monitoring and quantification methodol
+
+Once the outcome funding has been pledged, Ruari approaches several companies in the local Los Angeles region and gets buy-in from four which agree to start upgrading the schools per the bond specifications. These companies take on the upfront capital investment costs of the upgrades which, when completed at a particular location, are evaluated by either a representative of the specific school’s administration or the county school board. This representative presents the company with a signed verification that the school was updated to the specified standards. This signed verification is wrapped as part of the verified claim which Company A eventually bundles and submits as a claim group that confirms they have upgraded labs as confirmed by the site verification entity. 
+
+The verifiable claims submitted by Company A prove that they upgraded labs which will service 10,000 students and that this covers 35% of the students in East Los Angeles County. They receive the specified payout from the outcome fund based on this claim bundle for both the site specific and meta-level metrics they achieved. This project continues over the next 5 years with various companies joining and leaving the project. Five years later the project was able to get 80% of the labs upgraded in the set of schools initially scoped by the project and Ruari is now going out to try and raise another round of outcome funding with the intention of taking this widely successful program to more cities around the country. 
+
 
 #### User Needs
+User needs here are broken down for each of the three users presented in the above story: Ellenor, Vicki and Stephen. 
+
+__Outcome Funder__
+* Configuration of a clear set of criteria on which to base the outcome payments
+* Assurance that reported outcomes are properly validated by a clear and mutually agreed upon validation method
+* Mechanism to make the outcome payments as the various companies who are fronting the upfront costs reach desired outcome levels
+
+__Investing Companies__
+* Clear criteria for receiving outcome payments
+* Clear return on investment when the criteria for an outcome payment is met
```

Comment:
Financial returns on investment, which could include a coupon premium, performance bonus (though this is often passed through to the implementers), share of Impact Tokens generated - which is interesting in contexts where these tokens are tradable because there is demand and liquidity, or specific rights that can be accessed through owning the tokens (eg data access)